### ISSUE TITLE
[S3 Plugin] DTH S3 Plugin can not deploy in GovCloud due to missing assets

### ISSUE THREAD

**Issue by YikaiHu** [aws-solutions/data-transfer-hub#140]
**Describe the bug**
DTH S3 Plugin can not deploy in GovCloud due to missing assets. 

Should add aws-us-gov mapping in CloudFormation assets mappings.
```
"Mappings": {
    "FinderStackAssetTableC56DDAEC": {
      "aws": {
        "assetDomain": "https://aws-gcr-solutions-assets.s3.amazonaws.com/"
      },
      "aws-cn": {
        "assetDomain": "https://aws-gcr-solutions-assets.s3.cn-northwest-1.amazonaws.com.cn/"
      },
      "aws-us-gov": {
        "assetDomain": "https://aws-gcr-solutions-assets.s3.amazonaws.com/"
      }
    },
```

**To Reproduce**
Deploy S3 Plugin in GovCloud

**Expected behavior**
Deployment successful.

**Please complete the following information about the solution:**
- [ ] Version: [v2.6.0]

To get the version of the solution, you can look at the description of the created CloudFormation stack. For example, "_(SO8001) - Data Transfer Hub with aws-solutions-constructs: This template deploys an one-stop toolset for transferring data from different sources into AWS. Template version **v1.0.0**_".

- [ ] Region: [us-gov-west-1/us-gov-east-1]

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).

**Additional context**

**Comment by YikaiHu** [comment]
Workaround: please using this patching template.

[DataTransferS3Stack-gc-patch.json](https://github.com/awslabs/data-transfer-hub/files/14119982/DataTransferS3Stack-gc-patch.json)


We will fix this issue in next release.

**Comment by YikaiHu** [comment]
Fixed in v2.6.0
